usaspending_api/etl/management/commands/load_table_to_delta.py

```diff
@@ -75,14 +73,13 @@ def handle(self, *args, **options):
         spark.sql(f"use {destination_database};")
 
         # Resolve JDBC URL for Source Database
-        jdbc_url = os.environ.get(JDBC_URL_KEY)
-        jdbc_url = get_jdbc_url_from_pg_uri(jdbc_url)
+        jdbc_url = get_jdbc_url()
         if not jdbc_url:
-            raise RuntimeError(f"Looking for JDBC URL passed to env var '{JDBC_URL_KEY}', but not set.")
+            raise RuntimeError(f"Could'nt find JDBC url, please properly configure your CONFIG.")
         if not jdbc_url.startswith("jdbc:postgresql://"):
             raise ValueError("JDBC URL given is not in postgres JDBC URL format (e.g. jdbc:postgresql://...")
 
-        if partition_column_type == "numeric":
+        if partition_column_type in ("numeric", "long"):
```

Comment:
@collinwr is correcting the types that caused the need for this in his PR #3478. I incorrectly used `long` originally instead of `numeric` not realizing that this isn't talking about the actual type. Should be able to remove this change following that PR's merge into QAT.